client/layout/community-translator/launcher.jsx

```diff
@@ -28,6 +30,9 @@ class TranslatorLauncher extends React.PureComponent {
 		translate: PropTypes.func,
 	};
 
+	static translationScanner =
```

Comment:
I'm not sure entirely sure about this. It's nice to be able to catch all the startup translations, and this does get a lot of them, but I'm not sure if it's early enough to catch all of them.